share/wake/lib/system/remote_cache_runner.wake

```diff
@@ -59,6 +59,167 @@ export def mkRemoteCacheRunner (rscApi: RemoteCacheApi) (hashFn: Result RunnerIn
 
         Pass output
 
+    def rehydrateJob response label input job =
```

Comment:
This looks like a clean refactor into a new function, but I did read through it again just in case anything else stood out.